projects/addon-charts/components/line-chart/line-chart.component.ts

```diff
@@ -99,6 +109,21 @@ export class TuiLineChartComponent {
 
     value: readonly TuiPoint[] = [];
 
+    hovered$ = this._hovered$.pipe(distinctUntilChanged(), tuiZoneOptimized(this.ngZone));
+
+    hoveredOverAxes$ = this.hints$.pipe(map(values => !!values.length)).pipe(
+        switchMap(hasHints =>
+            (
+                this.axes?.hovered$ ??
+                this.hovered$.pipe(map(indexes => !Number.isNaN(indexes)))
+            ).pipe(map(hovered => hasHints || hovered)),
+        ),
+        debounceTime(100),
+        filter(visible => !visible),
+        tap(() => this.onHovered(NaN)),
```

Comment:
`hoveredOverAxes$` calls `this.onHovered`.
`this.onHovered`  => `this._hovered$.next(index)`.
`this._hovered$` => `hovered$`
`hovered$` => `hoveredOverAxes$`

Looks like possible cyclic problem

Reply:
fixed